tools/create_self_signed_cert.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -eu
+
+# Creates a self-signed certificate.
+#
+# For those with access, certificates are available in:
+#
+# https://github.com/oxidecomputer/configs/tree/master/nginx/ssl/wildcard.oxide-preview.com
```

Comment:
This location is will change eventually as a some things get decoupled. Not to block this PR, but for future reference.

Reply:
Thanks for the heads up. The flow for importing a non-self-signed cert is clearly half-baked; I needed to pull them manually.

Until we have a good authentication story for it, the manual approach seems better than something insecure, but I'd like to remedy that when we can.